src/ContainerRegistry/ContainerRegistry/Commands/ConnectAzureContainerRegistry.cs

```diff
@@ -60,6 +64,16 @@ protected override void InitDebuggingFilter()
 
             string LoginScript = string.Format("'{2}' | docker login {0} -u {1} --password-stdin", this.RegistryDataPlaneClient.GetEndPoint(), this.UserName, this.Password);
             WriteObject(this.ExecuteScript<object>(LoginScript));
+            if (ExposeToken) {
```

Comment:
This is a switch parameter. Should be
```suggestion
            if (ExposeToken.IsPresent) {
```